packages/terra-functional-testing/src/services/wdio-visual-regression-service/modules/getTerraFormFactor.js

```diff
@@ -1,20 +1,45 @@
-import { TERRA_VIEWPORTS } from '../../../constants';
+/* global browser */
+
+const { TERRA_VIEWPORTS } = require('../../../constants');
+
 /**
- * Determines the Terra form factor to for the current viewport size.
+ * Determines the current viewport size.
  *
- * @param {Number} viewpointWidth - Current width of the viewpoint.
- * @returns {String} - Terra form factor the current viewport with falls under.
+ * @returns {Object} - the current viewport size.
  */
-export default function getTerraFormFactor(viewpointWidth) {
+function getViewportSize() {
+  // eslint-disable-next-line prefer-arrow-callback
+  const resolution = browser.execute(function getResolution() {
+    return {
+      screenWidth: Math.max(document.documentElement.clientWidth, window.innerWidth || 0),
+      screenHeight: Math.max(document.documentElement.clientHeight, window.innerHeight || 0),
+    };
+  });
+
+  return {
+    width: resolution.screenWidth,
+    height: resolution.screenHeight,
+  };
+}
+
+/**
+ * Determines the Terra form factor for the current viewport size.
+ *
+ * @returns {String} - Terra form factor the current viewport falls under.
+ */
+function getTerraFormFactor() {
   const viewports = Object.entries(TERRA_VIEWPORTS);
+  const currentWidth = getViewportSize().width;
 
   for (let index = 0; index < viewports.length; index += 1) {
     const [formFactor, size] = viewports[index];
 
-    if (viewpointWidth <= size.width) {
+    if (currentWidth <= size.width) {
       return formFactor;
     }
   }
 
-  return 'enormous';
+  return 'huge';
```

Comment:
The largest viewport size is enormous without a max width. I think this should still be returned form factor based on the comparison logic

Reply:
That makes sense to me. Changed it back to `enormous`: https://github.com/cerner/terra-toolkit/commit/b85221c60247a8a943bdf916e2fb00df6cee06f7

My initial thinking was to default it to `huge` just like what we're doing [here](https://github.com/cerner/terra-toolkit/pull/545/files#diff-a6ba3c1d16c08e58e7ee3994f3ab541b63b2a3a8dafb0de9daed2c8090c948d9R94).